### PR TITLE
Enable test bench test in simple scripts and test in Travis CI

### PR DESCRIPTION
**Pull request by xiulipan**
Update the host-testbench scripts to test 3 simple testbench
volume, src and eqiir.
Enable Travis CI test for the testbench with this script.

### CODE REVIEW

File: tools/test/audio/comp_run.sh [hunk 1]
```diff
@@ -16,7 +16,7 @@ HOST_ROOT=../../testbench/build_testbench
 HOST_EXE=$HOST_ROOT/install/bin/testbench
 HOST_LIB=$HOST_ROOT/sof_ep/install/lib
 TPLG_LIB=$HOST_ROOT/sof_parser/install/lib
-TPLG_DIR=../../test/topology
+TPLG_DIR=../../build_tools/test/topology
```
Comment (singalsu) [inline tools/test/audio/comp_run.sh]: Oh, I see now that I've been generating the topologies into wrong directory. Thanks!

**Comment by xiulipan** [comment]
@lgirdwood I haven't test the Travis scripts yet, it seem some issue with the test toplogy build here.
I will send a fix. @singalsu I think your fix for testbench is merged?

**Comment by xiulipan** [comment]
@singalsu @lgirdwood updated and Travis test passed.

**Comment by singalsu** [comment]
I just checked, testbench works for me too.

**Comment by jajanusz** [comment]
> @zrombel any idea why Internal CI would fail here, I dont think it tests the testbench ?

I've reran this test, cos it looked like some random issue.

**Comment by lgirdwood** [comment]
CI known issue.

**Comment by marc-hb** [comment]
@singalsu , @xiulipan is there a plan for `host-testbench.sh` not to exit success 0 on failure? Until it does this interest of running it in CI seems limited.

```
./scripts/host-testbench.sh | grep -i fail; echo $?
du: cannot access 'volume_out.raw': No such file or directory
volume test failed!
du: cannot access 'volume_out.raw': No such file or directory
eqiir test failed!
0   #   <------
```

**Comment by xiulipan** [comment]
@marc-hb Good catch, I think I ignore some case here. Will `set -e` work here?

File: scripts/host-testbench.sh [hunk 2]
```diff
@@ -1,42 +1,91 @@
-#!/bin/sh
+#!/bin/bash
 # SPDX-License-Identifier: BSD-3-Clause
 # Copyright(c) 2018 Intel Corporation. All rights reserved.
```
Comment (xiulipan) [inline scripts/host-testbench.sh:4]: @marc-hb Add `set -e` here?

**Comment by marc-hb** [comment]
@xiulipan I filed #2752, we can discuss there.

In the mean time let's stop showing a green box in Travis: #2753 

> Have run shellcheck (apt-get install shellcheck) for the bash scripts? 

Apparently not..

File: .travis.yml [hunk 3]
```diff
@@ -74,3 +75,8 @@ jobs:
       env: PLATFORM=imx8x
     - <<: *qemuboottest
       env: PLATFORM=imx8m
+    - stage: testbench
+      script:
+        - ./scripts/docker-run.sh ./scripts/build-tools.sh -t &> /dev/null
```
Comment (marc-hb) [inline .travis.yml:80]: Please never ever discard all output like this, especially not in CI where the exact configuration can be difficult to reproduce locally. If the script is too verbose then its verbosity can be adjusted but discarding everything is throwing the baby with the bathwater.

Fixed in #3364

**Pull request merged**